index.html

```diff
@@ -1,15 +1,86 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
Aquí falta cambiar el idioma, la pagina no está en inglés
```suggestion
<html lang="es">
```